docs/editor/debugging.md

```diff
@@ -319,6 +319,27 @@ Source maps can be generated with two kinds of inlining:
 
 VS Code supports both the **inlined source maps** and the **inlined source**.
 
+#### JavaScript Source Map Tips
+

```

Comment:
I would like to see an example of how source maps need to be setup.
Maybe create a public git hub repositore as a sample and then just link it from here.